website/docs/containers/registries/index.md

```diff
@@ -2,18 +2,252 @@
+   1. Restart the Podman machine.
+
+   </TabItem>
+   <TabItem value="mac" label="macOS">
+
+   1. Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > Resources**.
+   1. Restart the Podman machine.
+
+   </TabItem>
+   <TabItem value="linux" label="Linux (rootless)">
+
+   - Stop all Podman processes.
+
+     ```shell-session
+     $ pkill podman
+     ```
+
+   </TabItem>
+   <TabItem value="linux-rootful" label="Linux (rootful)">
+
+   - Restart Podman.
+
+     ```shell-session
+     $ sudo systemctl restart podman
+     ```
+
+   </TabItem>
+   </Tabs>
+
+## Verifying your registry is properly configured
+
+To verify your registry has been properly configured, you can do the following steps:
+
+1. In **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**, the line with your registry has content in the Username and Password column, and action icons replacing the Configure button.
+
+1. Pull a private image from the registry.
+
+   1. Get the name of a private image stored in your registry, such as `quay.io/my-repository/my-image`, `ghcr.io/my-repository/my-image`, `docker.io/my-repository/my-image`, or `my-registry.tld/my-repository/my-image`.
+   1. Go to **Images**.
+   1. Click **Pull an image**.
+   1. On the **Image to Pull** screen:
+      1. **Image to pull**: Enter the image name.
+      1. Click Pull image.
+   1. Click Done.
+
+1. Push an image to the registry:
+
+   1. Get the fully qualified image name that your registry requires, such as `quay.io/my-repository/my-image`, `ghcr.io/my-repository/my-image`, or `docker.io/my-repository/my-image`.
+   1. Go to **Images**.
+   1. Click **Build an image**.
+   1. On the **Build Image from Containerfile** screen
+      1. **Containerfile path**: select the Containerfile or Dockerfile to build.
+      2. **Image Name**: enter the fully qualified image name that your registry requires.
+      3. Click Build.
+      4. Click Done.
+   1. On your image line, click **<Icon icon="fa-solid fa-ellipsis-v" size="lg" />**.
+
+      The contextual menu has a **Push Image to _your registry_** entry.
+
+## Changing your registry credentials
+
+To change your registry credentials, you can do the following steps:
+
+1. Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
+1. On your registry line, click **<Icon icon="fa-solid fa-ellipsis-v" size="lg" />**.
+1. Click **Edit password**.
+1. Enter your credentials in the **Username** and **Password** fields, and click **Login**.
+
+Podman Desktop logs Podman in with the updated credentials.
+
+## Removing a registry
+
+To remove your registry, you can do the following steps:
+
+1. Go to **<Icon icon="fa-solid fa-cog" size="lg" /> Settings > Registries**.
+1. On your registry line, click **<Icon icon="fa-solid fa-ellipsis-v" size="lg" />**.
+1. Click Remove.
+
+Podman Desktop removes the registry from the settings, and logs Podman out from the registry.
+
+## Understanding the registry configuration
```

Comment:
```suggestion
## The configuration files
```

Reply:
Not sure this one is clear. Maybe something like "Podman's registries configuration files" ?

Reply:
What about:

```suggestion
## Finding Podman registry configuration files
```

Reply:
Or: 
```suggestion
## Podman registry configuration files
```